lib/datasource/bitbucket-tags/index.ts

```diff
@@ -0,0 +1,65 @@
+import URL from 'url';
+import * as utils from '../../platform/bitbucket/utils';
+import * as packageCache from '../../util/cache/package';
+import { BitbucketHttp, bitbucketApiEndpoint } from '../../util/http/bitbucket';
+import { GetReleasesConfig, ReleaseResult } from '../common';
+
+const bitbucketHttp = new BitbucketHttp();
+
+export const id = 'bitbucket-tags';
+export const defaultRegistryUrls = [bitbucketApiEndpoint + '2.0/'];
```

Comment:
```suggestion
export const defaultRegistryUrls = [bitbucketApiEndpoint];
```
Not sure if we want `https://bitbucket.org` here? Do we need to support other hosts than `bitbucket.org`?
I don't think there are any other hosting options available´, so i prefer to simplify this.

Reply:
Well, there are Bitbucket Server, but I don't know if API is the same there.

Anyway, I'm a bit confused about this `defaultRegistryUrls`.
What do we want to put there? Should it be API URL (`api.bitbucket.org/2.0/repositories/org/repo`) our URL that is used in the browser (`bitbucket.org/org/repo`)?

Reply:
bitbucket server is a totally other product, with different api, so would be handled in separate datasource.

we use the public urls for gitlab and github, so i suggest to use `defaultRegistryUrls = ['https://bitbucket.org']`.
Then we simply ignore it and use relative urls, so it falls back to `api.bitbucket.org` in BitbucketHttp class